tests/test_pod.py

```diff
@@ -0,0 +1,185 @@
+# pylint: disable=missing-function-docstring,missing-module-docstring
+import json
+from pathlib import Path
+from typing import Union
+
+import pytest
+from _pytest.mark import ParameterSet
+
+from pytest_container.container import Container
+from pytest_container.container import DerivedContainer
+from pytest_container.container import PortForwarding
+from pytest_container.pod import Pod
+from pytest_container.pod import pod_from_pytest_param
+from pytest_container.pod import PodData
+from pytest_container.pod import PodLauncher
+from pytest_container.runtime import OciRuntimeBase
+from pytest_container.runtime import PodmanRuntime
+from tests.test_container_build import LEAP
+from tests.test_container_build import LEAP_WITH_MAN
+from tests.test_launcher import CONTAINER_THAT_FAILS_TO_LAUNCH
+from tests.test_port_forwarding import WEB_SERVER
+
+ALPINE = Container(
+    url="docker.io/library/alpine", custom_entry_point="/bin/sh"
```

Comment:
nit: I've seen the below used in [Podman](https://github.com/danishprakash/podman/blob/add-group-entry/test/e2e/config.go) too, essentially moving all image identifiers to a single package/file and using them across the test suite. I see that you're using this same string in multiple places for e.g. https://github.com/dcermak/pytest_container/blob/cdb1baffb1af2791fb626b89975596c27df90f3a/tests/test_container_build.py
```suggestion
    url=images.ALPINE, custom_entry_point="/bin/sh"
```